images/scorecard-test/cmd/test/main.go

```diff
@@ -35,7 +38,7 @@ import (
 // test image.
 
 const (
-	bundlePath = "/scorecard"
+	bundleZip = "/scorecard/bundle.zip"
```

Comment:
What is the bundle zip ?

Reply:
Could we add a comment explaining ?

Reply:
good catch, originally I was using zip and then switched to tar to archive up the bundle contents, I've updated this and added a comment.